pkgs/development/python-modules/pyqt/6.x.nix

```diff
@@ -103,7 +104,8 @@ buildPythonPackage rec {
     # ++ lib.optional withConnectivity qtconnectivity
     ++ lib.optional withMultimedia qtmultimedia
     ++ lib.optional withWebSockets qtwebsockets
-    ++ lib.optional withLocation qtlocation;
+    ++ lib.optional withLocation qtlocation
+    ++ lib.optional withPdf qtwebengine;
```

Comment:
```suggestion
    ++ lib.optional withPdf qpdf;
```
is not sufficient?

Reply:
Never mind, I just learned qtpdf != qpdf.

Reply:
:)